crates/wasmtime/src/config.rs

```diff
@@ -414,10 +429,15 @@ impl Config {
             .set("enable_probestack", "false")
             .expect("should be valid flag");
 
+        // Reference types are enabled by default, so enable safepoints
+        flags
+            .set("enable_safepoints", "true")
+            .expect("should be valid flag");
```

Comment:
We may actually want to avoid this to avoid duplicating the logic below, and instead we could initialize `ret` like `wasm_backtrace_details` is initialized perhaps?

Reply:
(ideally the same could be done for `opt_level` too)